client/src/pages/AddRide.js

```diff
@@ -1,13 +1,14 @@
 import React, { useState } from "react";
 import styled from "styled-components/macro";
-import Badge from "../components/Badge";
-import { Card } from "../components/Card";
-import { HeaderMain } from "../components/HeaderMain";
-import { Input } from "../components/Input";
-import { Button } from "../components/Button";
-import { addRide } from "../utils/api";
 import { useHistory } from "react-router-dom";
 
+import Badge from "../components/Badge";
+import Card from "../components/Card";
+import HeaderMain from "../components/HeaderMain";
+import Input from "../components/Input";
+import Button from "../components/Button";
+import addRide from "../utils/api";
```

Comment:
Change `import addRide from "../utils/api"` to `import { addRide } from "../utils/api"`

Reply:
already changed ... CI is faster than you are 👩‍🎤